godotproject/Level1.tscn

```diff
@@ -352,6 +352,8 @@ player_nodepath = NodePath("../Player")
 
 [node name="DeathPlane" parent="." instance=ExtResource( 9 )]
 position = Vector2( -56, -680 )
+level0_start_postion = Vector2( 121, 896 )
+level1_start_postion = Vector2( 121, 256 )
```

Comment:
In Level1.tscn, set the start_position to the correct value for Level 1 (121, 256)
```suggestion
[node name="DeathPlane" parent="." instance=ExtResource( 9 )]
position = Vector2( -56, -680 )
start_position = Vector2( 121, 256 )
```